packages/ui-components/src/components/svg-icon/svg-icon.scss

```diff
@@ -0,0 +1,107 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	 * @prop --icon-color: Color of the icon
+	 * @prop --icon-height: Height of icon
+	 * @prop --icon-width: Width of icon
+	 * @prop --icon-rotation: Rotation of icon
+	 */
+	--icon-color: kv-color('text');
+	--icon-height: 16px;
+	--icon-width: 16px;
+	--icon-rotation: 0deg;
+}
+
+.svg-icon {
+	/**
+	 * @prop --illustration-color: Base color used in the illustration
+	 * @prop --illustration-color-light-90: 90% lighter version of the illustration base color
+	 * @prop --illustration-color-light-80: 80% lighter version of the illustration base color
+	 * @prop --illustration-color-dark-55: 55% darker version of the illustration base color
+	 * @prop --illustration-color-dark-35: 35% darker version of the illustration base color
+	 * @prop --illustration-success-color: Success color used in the illustration
+	 */
+	--illustration-color: var(--kv-illustrations-color, #{$primary-color});
+	--illustration-color-light-90: var(--kv-illustrations-color-light-90, #{mix(#fff, $primary-color, 89.5%)});
+	--illustration-color-light-80: var(--kv-illustrations-color-light-80, #{mix(#fff, $primary-color, 78.5%)});
+	--illustration-color-dark-55: var(--kv-illustrations-color-dark-55, #{mix(#000, $primary-color, 54.5%)});
+	--illustration-color-dark-35: var(--kv-illustrations-color-dark-35, #{mix(#000, $primary-color, 36%)});
+	--illustration-success-color: var(--kv-illustrations-success-color, #{$success-color});
```

Comment:
Why are the illustration props on the `.svg-icon` selector instead of the `:host`?